event.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
The JSON structure should include a top-level key that describes the context of the data, such as `event` or `payload`, to improve clarity and organization.